src/account/AccountUpdateTransaction.js

```diff
@@ -265,9 +265,7 @@ export default class AccountUpdateTransaction extends Transaction {
                     update.stakedNodeId != null
                         ? update.stakedNodeId
                         : undefined,
-                declineStakingReward:
-                    update.declineReward != null &&
-                    update.declineReward == true,
+                declineStakingReward: update.declineReward != null,
```

Comment:
Why we are removing the following check `update.declineReward == true`?

Reply:
Due to this error:
![Screenshot 2023-01-20 at 18 44 51](https://user-images.githubusercontent.com/13587218/213755125-d5224ab0-d190-4aa1-b7de-69a0ba3622c1.png)

I am still figuring it out.